<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Deposit</class>
 <widget class="QMainWindow" name="Deposit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>700</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DepositWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">padding:5px;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>500</width>
     <height>700</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>400</width>
     <height>400</height>
    </size>
   </property>
   <widget class="QRadioButton" name="flag">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>540</y>
      <width>150</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>interest capitalization</string>
    </property>
   </widget>
   <widget class="QLabel" name="rate">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>150</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Interest rate</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_rate">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>250</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>1.50</string>
    </property>
   </widget>
   <widget class="QLabel" name="deposit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>150</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Deposit amount</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_deposit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>250</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>1000</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_period">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>250</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>6</string>
    </property>
   </widget>
   <widget class="QLabel" name="period">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>150</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Placement period</string>
    </property>
   </widget>
   <widget class="QComboBox" name="dropdown_replemishment">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>240</y>
      <width>250</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>250</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>220</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
    </property>
    <item>
     <property name="text">
      <string>Without replenishment</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Replenishment once a month</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Replenishment quarterly</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Replenishment every six months</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Replenishment once a year</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="input_withdraw">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>300</y>
      <width>150</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>0</string>
    </property>
   </widget>
   <widget class="QComboBox" name="dropdown_withdraw">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>300</y>
      <width>250</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>220</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>250</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <item>
     <property name="text">
      <string>Without removal</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Withdrawal once a month</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Withdrawal once a quarter</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Withdrawal every six months</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Withdrawal once a year</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="input_replenishment">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>240</y>
      <width>150</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="output_deposit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>420</y>
      <width>150</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Deposit amount (end)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_percent">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>360</y>
      <width>250</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_tax">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>480</y>
      <width>250</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="output_percent">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>360</y>
      <width>150</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Accrueds</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_depositEnd">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>420</y>
      <width>250</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="output_tax">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>480</y>
      <width>150</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tax amount</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="calculate">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>540</y>
      <width>250</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
   <widget class="QLabel" name="rate_tax">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>150</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tax rate</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_taxrate">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>180</y>
      <width>250</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>13.0</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
